neware_parser/plot.py

```diff
@@ -376,6 +382,8 @@ def plot_predicted(
             )
         elif mode == "cv":
             pred_cap = test_results["pred_cv_capacity"].numpy()[:, -1]
```

Comment:
there is some tensorflow here. this is plotting code. 
there shouldn't be any tensorflow here.
This should be in the data engine